packages/xo-web/src/common/xo/register-proxy-modal/index.js

```diff
@@ -0,0 +1,60 @@
+import _ from 'intl'
+import Component from 'base-component'
+import Icon from 'icon'
+import React from 'react'
+import SingleLineRow from 'single-line-row'
+import { Col, Container } from 'grid'
+import { Input as DebounceInput } from 'debounce-input-decorator'
+
+export default class RegisterProxyModal extends Component {
+  state = {
+    address: undefined,
+    authenticationToken: undefined,
+    name: undefined,
+    vmUuid: undefined,
+  }
+  get value() {
+    return this.state
+  }
+
+  render() {
+    const { address, authenticationToken, name, vmUuid } = this.state
+    return (
+      <Container>
+        <SingleLineRow className='mt-1'>
+          <Col size={6}>{_('proxyAuthToken')}</Col>
+          <Col size={6}>
+            <DebounceInput
+              className='form-control'
+              onChange={this.linkState('authenticationToken')}
+              value={authenticationToken}
```

Comment:
Isn't there an issue when passing `undefined` to input components' `value`?

Doesn't this make them uncontrolled?